src/components/Pages/SetupPage/SetupPage.tsx

```diff
@@ -79,17 +81,52 @@ export const SetupPage = () => {
     return ''
   }, [agree])
 
-  const handleRandomize = useCallback(() => setProfile(getRandomDefaultProfile()), [])
-  const handleContinue = useCallback(() => setView(View.FORM), [])
-  const handleBack = useCallback(() => setView(View.RANDOMIZE), [])
+  // Sets a random default profile.
+  const handleRandomize = useCallback(() => {
+    getAnalytics().track(TrackingEvents.CLICK, {
+      action: ClickEvents.RANDOMIZE
+    })
+
+    setProfile(getRandomDefaultProfile())
+  }, [])
+
+  // Confirms the current default profile and goes to the form view.
+  const handleContinue = useCallback(() => {
+    getAnalytics().track(TrackingEvents.AVATAR_EDIT_SUCCESS, {
+      // eslint-disable-next-line @typescript-eslint/naming-convention
```

Comment:
Ignoring this rule as analytic events use a different casing than in our dapp standards